catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -332,6 +332,12 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
         String attributeQuery = getQuery(attributeName, identifiers);
 
         SolrQuery query = new SolrQuery(attributeQuery);
+        // Set number of rows to the result size + 1.  The default row size in Solr is 10, so this
```

Comment:
I thought we were avoiding inline comments? How about pulling this line of code into its own method and turn the comments into the method's javadoc?